docs/api/management.oas3.yaml

```diff
@@ -0,0 +1,342 @@
+
+    <small>Conforming Response</small>
+    ```json
+    {
+      "@id": "<x>",
+      "skos:prefLabel": [
+        {
+          "@lang": "en",
+          "@value": "test"
+        }
+      ]
+    }
+    ```
+
+    <small>Non-conforming Response</small>
+    ```json
+    {
+      "@id": "<x>",
+      "skos:prefLabel": "test"
+    }
+    ```
+
+    # Deferencing Resources
+
+    It **MUST** be possible to deference all resources that are referenced by IRI. For example, when a GET request to a _skos:Concept_
+    provides a list of IRIs that are _broader_ or _narrower_ concepts then each of those elements should have an IRI that dereferences to
+    a taxonomy management endpoint. This is in addition to providing all [Lexical labels](https://www.w3.org/TR/skos-reference/#labels) for each resource
+    as described in the Data Model.
+
+
+tags:
+  - name: Project
+    description: |
+      A project is a collection of Concept Schemes, and some associated metadata. A project does not own concept schemes,
+      however can associate itself with them by using the `dcterms:hasPart` term.
+  - name: ConceptScheme
+    description: |
+      A Concept Scheme contains a collection of concepts, with several of as top-level concepts that serve as an entrypoint into the taxonomy
+      graph.
+  - name: Concept
+    description: Operations executed on single concepts.
+paths:
+  /0.1/project/{project}:
+    get:
+      parameters:
+        - $ref: '#/components/parameters/project'
+      summary: Get a project
+      tags:
+        - Project
+      responses:
+        '201':
+          description: Compact JSON-LD representation of a Project resource.
+          content:
+            'application/ld+json;profile="http://www.w3.org/2004/02/skos/core.jsonld"':
+              example:
+                $ref: './samples/project/new-project--response.json'
+              schema:
+                $ref: './schemas/project/response.json'
+    put:
+      summary: Update a project
+      tags:
+        - Project
+      requestBody:
+        $ref: '#/components/requestBodies/project'
+      responses:
+        '422':
+          $ref: '#/components/responses/validationFailed'
+        '204':
+          description: Update successful. Resource is unchanged from reqest body.
+
+  /0.1/project:
+    post:
+      summary: Create a project
+      tags:
+        - Project
+      requestBody:
+        $ref: '#/components/requestBodies/project'
+      responses:
+        '422':
+          $ref: '#/components/responses/validationFailed'
+        '201':
+          description: JSON-LD representation of the new Project resource.
+          content:
+            'application/ld+json;profile="http://www.w3.org/2004/02/skos/core.jsonld"':
+              example:
+                $ref: samples/project/new-project--response.json
+              schema:
+                $ref: schemas/project/response.json
+
+  /0.1/concept-scheme/{scheme}/concept/{concept}:
```

Comment:
Both, really. This is something I've touched on briefly with @stephenwf. The frontend currently uses the latter, but the former is the canonical URI for a concept.

Originally this was written with the intention of a concept being associated exclusively to exactly 1 concept scheme, but that's no longer the case. Since a concept can exist across multiple schemes, a way to address that concept unambiguously is needed which is why we have `/v0.1/concept/{concept}`.

For the sake of ease of use, while integrating with the frontend we'll add `/0.1/concept-scheme/{scheme}/concept/{concept}`, which will be functionally equivalent with the addition of attaching the new concept to the concept scheme as a top concept.

(cc @stephenwf).